# serial_msg.msg

uint32 id
uint32 length
uint32 data_type

int32   data_int
float32 data_float

